a.txt

```diff
@@ -0,0 +1 @@
+this is a file a.txt created by Rutuja user
```

Comment:
please write onemore line of code
thanks
Rishi

Reply:
added one line as suggested by Rishi

Reply:
Good, PR is approved